index.html

```diff
@@ -1,15 +1,59 @@
 <!DOCTYPE html>
 <html lang="en">
+
```

Comment:
The spacing is being used for readability and separating different parts of the page, this is fine for me. Just be mindful that some developers might prefer for there to be no empty spaces.